src/main/java/enmasse/kafka/bridge/tracker/SimpleOffsetTracker.java

```diff
@@ -109,9 +109,10 @@ public synchronized void commit(Map<TopicPartition, OffsetAndMetadata> offsets)
 			
 				// if offset tracked isn't changed during Kafka committing operation 
 				// (it means no other messages were acknowledged)
-				if (this.offsets.get(offset.getKey().partition()) == offset.getValue().offset()) {
+				PartitionState x = this.offsets.get(offset.getKey().partition());
```

Comment:
Can we use something like "state" or "partitionState" as variable name instead of just "x" ?